src/DatasetTree.ts

```diff
@@ -98,14 +98,15 @@ export class DatasetTree implements IZoweTree<ZoweNode> {
                     node.iconPath = applyIcons(node);
                     this.mFavorites.push(node);
                 } catch(e) {
-                    vscode.window.showErrorMessage(
-                        localize("initializeFavorites.error.profile1",
-                        "Error: You have Zowe Data Set favorites that refer to a non-existent CLI profile named: ") + sesName +
-                        localize("intializeFavorites.error.profile2",
-                        ". To resolve this, you can create a profile with this name, ") +
-                        localize("initializeFavorites.error.profile3",
-                        "or remove the favorites with this profile name from the Zowe-DS-Persistent setting, which can be found in your ") +
-                        getAppName(extension.ISTHEIA) + localize("initializeFavorites.error.profile4", " user settings."));
+                    const errMessage: string =
+                    localize("initializeFavorites.error.profile1",
+                    "Error: You have Zowe Data Set favorites that refer to a non-existent CLI profile named: ") + sesName +
+                    localize("intializeFavorites.error.profile2",
+                    ". To resolve this, you can create a profile with this name, ") +
+                    localize("initializeFavorites.error.profile3",
+                    "or remove the favorites with this profile name from the Zowe-DS-Persistent setting, which can be found in your ") +
```

Comment:
We should be mindful when separating localized messages.
I assume the intention is to reuse the text in other places, but it might be hard to assume that every language is structured the same as english. (e.g. Say we translate the extension to Japanese one day, these two messages may be written differently to provide a more natural/conversational message to the end-user)
I understand this was done before this PR, just something to think about for future reference.